packages/affine/shared/src/adapters/type.ts

```diff
@@ -17,16 +17,17 @@ export const isBlockSnapshotNode = (node: unknown): node is BlockSnapshot =>
 
 export type AdapterContext<
   ONode extends object,
-  TNode extends object,
-  TConverter extends DeltaASTConverter,
+  TNode extends object = never,
+  TConverter extends DeltaASTConverter = DeltaASTConverter,
 > = {
   walker: ASTWalker<ONode, TNode>;
   walkerContext: ASTWalkerContext<TNode>;
   configs: Map<string, string>;
   job: Job;
-  deltaConverter: TConverter;
+  deltaConverter?: TConverter;
```

Comment:
For delta converter, maybe set a dummy converter for plain-text to make it non-optional?

Reply:
That make sense